src/scene/model/vertex.rs

```diff
@@ -28,6 +31,8 @@ impl Default for Vertex {
             normal: Zero::zero(),
             tangent: Zero::zero(),
             tex_coords: Zero::zero(),
+            #[cfg(feature = "vertex-color")]
+            color: Vector4::new(0, 0, 0, 0),
```

Comment:
```suggestion
            color: Zero::zero(),
```

Reply:
I found it more meaningful to use white (one) instead of transparent (zero) as default value here, since glTF specifies that the vertex color serves acts as an additional linear multiplier to base color.

Here I also tried `One::one`, but the trait `One` stands for "mathematical identity of that type" and is not implemented for vectors.